Formula/s/swift-section.rb

```diff
@@ -0,0 +1,44 @@
+class SwiftSection < Formula
+  desc "CLI tool for parsing mach-o files to obtain Swift information"
+  homepage "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection"
+  url "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection/archive/refs/tags/0.4.1.tar.gz"
+  sha256 "afe48172e28e7ee626edf051d58662b62b63ddb6780e830cafbe886007af8029"
+  license "MIT"
+  head "https://github.com/MxIris-Reverse-Engineering/MachOSwiftSection.git",
+       branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  # The Package.swift file requires Swift 5.10 or later.
+  # But it is actually only builable with Swift 6.1+ due to the usage of trailing commma in comma-separated lists.
+  depends_on xcode: ["16.3", :build]
+  depends_on :macos
+
+  uses_from_macos "swift" => :build
```

Comment:
```suggestion
  depends_on :macos

  uses_from_macos "swift" => :build, since: :sonoma # swift 5.10+
```

Reply:
Why `since: :sonoma` is needed? It is only requesting a Xcode 16.3 toolchain with Swift 6.1 compiler for me. There is too much limitation on the OS. (Currently it can be run on macOS 11+)